web/packages/teleport/src/Navigation/NavigationItem.tsx

```diff
@@ -195,6 +199,28 @@ export function NavigationItem(props: NavigationItemProps) {
         </Link>
       );
     }
+
+    if (isLocked(ctx.lockedFeatures)) {
+      if (!lockedNavigationItem || !lockedRoute) {
+        console.error('locked feature without an alternative route');
+        return null;
+      }
+
+      return (
+        <Link
+          {...linkProps}
+          onKeyDown={handleKeyDown}
+          tabIndex={props.visible ? 0 : -1}
+          to={lockedNavigationItem.getLink(clusterId)}
+          exact={lockedNavigationItem.exact}
+        >
+          <LinkContent size={props.size}>
+            {getIcon(props.feature, props.size)}
+            {lockedNavigationItem.title}
+          </LinkContent>
+        </Link>
+      );
+    }
```

Comment:
I think we don't have to copy the same JSX here, we could just do something like this:
```tsx
        let navigationItemVersion = navigationItem;
        if (isLocked(ctx.lockedFeatures)) {
            if (!lockedNavigationItem || !lockedRoute) {
                console.error('locked feature without an alternative route');
                return null;
            }
            navigationItemVersion = lockedNavigationItem;
        }

        return (
            <Link
                {...linkProps}
                onKeyDown={handleKeyDown}
                tabIndex={props.visible ? 0 : -1}
                to={navigationItemVersion.getLink(clusterId)}
                exact={navigationItemVersion.exact}
            >
                <LinkContent size={props.size}>
                    {getIcon(props.feature, props.size)}
                    {navigationItemVersion.title}
                </LinkContent>
            </Link>
        );
    }
```
(sorry for ugly formatting)